### PR TITLE
Consistent input/output from unit_converter, some EMIT fixes

### PR DESCRIPTION
**Pull request by myoung301**
### units:
- The unit_converter function was returning a non-list when a list was passed with one entry. The change makes the output type match the input type (list in = list out, single element in = single element out).
- Improved the warning message to include the unrecognized units when unrecognized units are passed to unit_converter.

### EMIT:
- Emit inherited from FieldAnalysisEmit, but EMIT doesn't have any need for the FieldAnalysis functionality (no analysis setups in EMIT design). Emit will now inherit directly from Design. (S776155)
- Emit.modeler.components is now initialized automatically (refresh_all_ids is called)
- Units weren't initialized before the modeler so refresh_all_ids would fail during Emit initialization. Moved units initialization ahead of modeler initialization.

### CODE REVIEW

**Comment by myoung301** [comment]
CI_Prerelease: https://github.com/pyansys/pyaedt/actions/runs/4256855860/jobs/7406255230

File: pyaedt/emit.py [hunk 1]
```diff
@@ -154,6 +159,31 @@ def __init__(
 
             self.__emit_api_enabled = True
 
+        # Update modeler IDs now, after units are initialized.
+        # self.modeler.components.refresh_all_ids()
```
Comment (jsalant22) [inline pyaedt/emit.py:163]:
```suggestion
```
Can we remove these?

**Review by jsalant22 — APPROVED** [review]

File: pyaedt/emit.py (hunk 1 above)
Comment (myoung301) [reply pyaedt/emit.py]: @jsalant22 - Yes! Good catch. Thank you.

**Comment by myoung301** [comment]
There is one test failure:
image

This is due to an update to the Emit API that was merged 2/21 so the CI_Prerelease workflow hasn't caught up yet.

@apollo-crum, @jsalant22 FYI - in case you run that workflow.

**Pull request merged**